src/components/App.tsx

```diff
@@ -2987,6 +2974,46 @@ class App extends React.Component<AppProps, AppState> {
     invalidateContextMenu = true;
   };
 
+  handleHoverSelectedLinearElement(
+    scenePointerX: number,
+    scenePointerY: number,
+  ) {
+    const element = LinearElementEditor.getElement(
+      this.state.selectedLinearElement!.elementId,
+    )!;
```

Comment:
instead of asserting this, pass it as an arg to `handleHoverSelectedLinearElement`

Reply:
that is, asserting `this.state.selectedLinearElement`.

As for asserting the `getElement()` return value, that we shouldn't do. Check instead. Asserting we should never do in general unless we need to. I've just run into an issue where the element wasn't defined.